backend/chalice/api_server/app.py

```diff
@@ -80,12 +80,15 @@ def dispatch(*args, **kwargs):
         # incompatible with the werkzeug.MultiDict expected by Flask.
         query_string = list(app.current_request.query_params.items()) if app.current_request.query_params else None
 
+        # set dummy auth token value for optional security endpoints
+        headers = [*app.current_request.headers.items(), ('cxgdummy', 'dummy')]
```

Comment:
since we need to document our cookie usage in cookie policy page, maybe we need to pick a more PC name lol? Like `cxgpublic = public` or something better 😆

Reply:
sounds good